pkg/redshift/driver/rows.go

```diff
@@ -183,7 +178,7 @@ func (r *Rows) Close() error {
 func (r *Rows) fetchNextPage(token *string) error {
 	var err error
 
-	r.result, err = r.service.GetStatementResult(&redshiftdataapiservice.GetStatementResultInput{
+	r.result, err = r.service.GetStatementResult(context.TODO(), &redshiftdata.GetStatementResultInput{
```

Comment:
We could where it's called from `newRows` but not from what I think is the main user - the `Next` method on line 43 here, which implements an interface from `sql/driver`. Although - we could do the not-ideal thing of storing the context on our `Rows` struct. Since the `Rows` struct is tied to a particular query ID that's probably not a bad idea here...